src/components/MarkdownTable.tsx

```diff
@@ -6,7 +6,12 @@ export interface IProps {
 }
 
 const MarkdownTable: React.FC<IProps> = ({ children }) => (
-  <Box my={8}>
+  <Box
+    my={8}
+    sx={{
```

Comment:
We actually don't need the sx prop here: https://chakra-ui.com/docs/styled-system/the-sx-prop
```
sx={{
    overflowX: "auto",
  }}
```

We can just use `overflowX` as a prop style instead here. Ill push that up.